app/jobs/rest_of_world_zones/rest_of_world_zone_create_job.rb

```diff
@@ -0,0 +1,76 @@
+module RestOfWorldZones
+  class RestOfWorldZoneCreateJob < ApplicationJob
+    def perform(shopify_shipping_zone)
+      create_rest_of_world_zones(shopify_shipping_zone)
+    end
+
+    def create_rest_of_world_zones(shopify_shipping_zone)
+      @shopify_shipping_zone = shopify_shipping_zone
+      @existed_spree_zones_by_member_type = existed_spree_zones_by_member_type
+      create_rest_of_world_country_zone
+    end
+
+    def existed_spree_zones_by_member_type
+      profile_id = @shopify_shipping_zone.profile_id.split('/').last
+      Spree::Zone.where("name like ?", "%#{profile_id}%").map(&:members).flatten.group_by{|m| m.zoneable_type}
+    end
+
+    def create_rest_of_world_country_zone
+      @rest_of_world_country_zone = Spree::Zone.create(
+        name: 'Rest of World - Countries',
+        description: 'shopify shipping to Countries Rest of World',
+        kind: 'country'
+      )
+      create_rest_of_world_state_zone if @existed_spree_zones_by_member_type.include?("Spree::State")
```

Comment:
Why?

Reply:
just because to understand that `include?` is checking keys in hash you should spend some time, but when you see `has_key?` you can just read the code as english sentence.
this one is minor and sure you can leave it as is